e2e/tests/services.rs

```diff
@@ -131,24 +128,18 @@ impl OrderbookServices {
         let db = Database::new("postgresql://").unwrap();
         db.clear().await.unwrap();
         let event_updater = Arc::new(EventUpdater::new(gpv2.settlement.clone(), db.clone(), None));
-        let current_block_stream = current_block_stream(web3.clone(), Duration::from_secs(1))
-            .await
-            .unwrap();
         let pair_provider = Arc::new(UniswapPairProvider {
             factory: uniswap_factory.clone(),
             chain_id,
         });
-        let pool_fetcher = CachedPoolFetcher::new(
-            Box::new(PoolFetcher {
-                pair_provider,
-                web3: web3.clone(),
-            }),
-            current_block_stream,
-        );
+        let pool_fetcher = OrderbookPoolFetcher::new(Box::new(PoolFetcher {
```

Comment:
Is this maybe missing parameters? Otherwise I'm a little confused.

Also, does the pool fetcher need to be added to the list of `maintiners` in the `ServiceMaintenance` instance :point_down:?

Reply:
I didn't finish integrating it fully which is why it's a draft. Just wanted to compare the different cache implementations.